core/node/proof_data_handler/Cargo.toml

```diff
@@ -17,6 +17,8 @@ zksync_dal.workspace = true
 zksync_object_store.workspace = true
 zksync_prover_interface.workspace = true
 zksync_types.workspace = true
+zksync_vm_executor.workspace = true
```

Comment:
Off-topic: This dependency is unfortunate given that `vm_executor` transitively depends on all VM implementations. The assumption behind including `L1BatchParamsProvider` in `vm_executor` was that its users would also execute VM based on returned `(SystemEnv, L1BatchEnv)`, but apparently this is not the case here.

Reply:
No clue how to fix this..

Reply:
No need to fix it rn; I'll probably create a dedicated task for it. Abstractly, it could make sense to place `L1BatchParamsProvider` elsewhere, but where exactly isn't clear to me yet (maybe, `zksync_state`, but it would look sort of out of place).